docs/kernel/modify_kernel.md

```diff
@@ -2,105 +2,183 @@
-time="2021-02-05T11:16:15-08:00" level=debug msg="+++ current_config\t2021-02-05 19:16:09.440117553 +0000"
-time="2021-02-05T11:16:15-08:00" level=debug msg="@@ -6484,8 +6484,6 @@"
-time="2021-02-05T11:16:15-08:00" level=debug msg=" # end of Compile-time checks and compiler options"
-time="2021-02-05T11:16:15-08:00" level=debug msg=" "
-time="2021-02-05T11:16:15-08:00" level=debug msg=" CONFIG_MAGIC_SYSRQ=y"
-time="2021-02-05T11:16:15-08:00" level=debug msg="-CONFIG_MAGIC_SYSRQ_DEFAULT_ENABLE=0x1"
-time="2021-02-05T11:16:15-08:00" level=debug msg="-CONFIG_MAGIC_SYSRQ_SERIAL=y"
-time="2021-02-05T11:16:15-08:00" level=debug msg=" CONFIG_DEBUG_KERNEL=y"
-time="2021-02-05T11:16:15-08:00" level=debug msg=" CONFIG_DEBUG_MISC=y"
-time="2021-02-05T11:16:15-08:00" level=debug msg=" "
-```
+Once your environment is prepared and the sources are present, you can  make your modifications to the relevant config files.  
+* For `x86_64`, modify the `config` file.  
+* For `AARCH64`, modify the `config_aarch64` file.  
 
-After editing your config file, save it and compute a new sha256sum.
+Currently, the `CONFIG_BLK_WBT` setting is disabled by default. For this tutorial, you will enable it. Run the following command to set `CONFIG_BLK_WBT` or manually edit the `config` file. 
 
 ```bash
-$ sha256sum config
-f6c3c5eb536f7c7778c3aaa45984de9bf6c58d2a7e5dfd74ace203faabf090a6  config
+# Enable CONFIG_BLK_WBT
+sed -i 's/# CONFIG_BLK_WBT is not set/CONFIG_BLK_WBT=y/' config
 ```
 
-Now, using your favorite editor update the config file hash(es) in the kernel.signatures.json.
+Ensure your change is in config and saved. Next, update the signature for `config`.
 
-One last step before building.  When there is a conflict, the build system will make a best-effort attempt at prioritizing the local version of a package over the version on packages.microsoft.com.  However, to ensure we can differentiate our new custom kernel from the default kernel, and to guarantee the local version will be consumed, bump the release number in the kernel release spec. In this case use your favorite editor and change the release number to 100 as shown below and save the file.
+```bash
+# Get the hash for the config file
+CONFIGHASH=$(sha256sum config | awk '{print $1}')
 
+# Update the hash for the config in the signatures file
+sed -i 's/    "config": .*/    "config": "'"$CONFIGHASH"'",/' kernel.signatures.json
 ```
+
+When there is a conflict, the build system will make a best-effort attempt at prioritizing the local version of a package over the version on [packages.microsoft.com](http://packages.microsoft.com/).  However to ensure you can differentiate your new custom kernel from the default kernel and to guarantee the local version will be consumed, bump the release number in the kernel release spec. In this case, use your favorite editor and change the release number to 100 as shown below and save the file.
```

Comment:
What happens when a newer version of the kernel gets published to packages.microsoft.com? Do our prioritization rules pick the latest version on PMC or the local bumped release number version?